plugins/main/loopback/loopback.go

```diff
@@ -49,6 +50,26 @@ func parseNetConf(bytes []byte) (*types.NetConf, error) {
 	return conf, nil
 }
 
+func GetRootNetNS() (netns.NsHandle, error) {
```

Comment:
/lgtm
But I think it's better to move these two functions to [https://github.com/containernetworking/plugins/blob/main/pkg/ns/ns_linux.go](https://github.com/containernetworking/plugins/blob/main/pkg/ns/ns_linux.go), seems better

Reply:
I have talked with squeed and I think it‘s better to fix it in `skel` for all plugins. Let's close it.